library/ingest.py

```diff
@@ -111,6 +116,16 @@ def wrapper(self, *args, **kwargs) -> list:
                 def update_progress(complete, message, unknown):
                     progress.update(task, completed=floor(complete * 1000))
 
+                # This addresses a gdal issue where translation will fail
+                # to generate a csv if one already exists at the dest path
```

Comment:
Does the translation for this case fail even with `"OVERWRITE=YES"` as an option for destination in the template yml?

And if the answer is no, seems like we'd want that always supplied, I can't think of a case where we wouldn't want to overwrite the local destination file

Reply:
Gotcha, weird. No, this seems fine!